### PR TITLE
Updates for SSE Integration:

### PR DESCRIPTION
**Pull request by stefaniasava**
**Adapting the frontend to  lisen for all SSE events and trigger the necessary reloads**

Backend SSE : UPdated sse.py to return a listener
Frontend SSE Handling:  Established SSE connection  in sse.js. 
Code Cleanup: Prettier modified some files
Refactored ReportItem class to use the updated report_item_updated method for SSE event triggering

## Summary by CodeRabbit

- **Documentation**
	- Updated README files with minor modifications for clarity in development sections and typo corrections.
- **New Features**
	- Enhanced SSE (Server-Sent Events) functionality for more efficient message streaming.
- **Bug Fixes**
	- Adjusted SVG element attributes in presenter templates for disinformation and OSINT to improve visualization.
	- Improved error handling and reconnection logic in SSE connections.
- **Refactor**
	- Optimized code by removing unnecessary trailing commas and empty lines in various Vue components and JavaScript stores.
	- Improved readability and presentation in GUI components by reformatting and reordering elements.
- **Style**
	- Modified visual presentation aspects in GUI components without altering core functionality.
- **Tests**
	- Added whitespace modifications in test HTML content for better readability.

### CODE REVIEW

**Comment by not4Pedro** [comment]
@coderabbitai review

File: src/core/pyproject.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ dependencies = [
     "python-dotenv",
     "gunicorn",
     "psycogreen",
-    "psycopg2",
+    "psycopg2-binary",
```
Comment (b3n4kh) [inline src/core/pyproject.toml:30]: For production psycopg2 is the recommended package - see: https://pypi.org/project/psycopg2-binary/

File: src/gui/src/utils/sse.js [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
+  const evtSource = new EventSource(sseEndpoint)
+
+  evtSource.onopen = () => console.debug('SSE connection opened.')
+  evtSource.onerror = (event) => console.error('SSE connection error:', event)
+  /*evtSource.onerror = (event) => {
+    console.error('SSE connection error:', event)
+    // Attempt to reconnect
+    evtSource.close(); // Close current connection
+    setTimeout(connectSSE, 5000) // Attempt reconnecting after some delay
+  }*/
```
[the remark this answers is not in the document]
Comment (b3n4kh) [reply src/gui/src/utils/sse.js:25]: Please check if the commented code can be either used or deleted

File: src/gui/src/utils/sse.js [hunk 3]
```diff
@@ -1,33 +1,60 @@
...
+
+  this.sseConnection = evtSource
 }
 
 export function reconnectSSE() {
-  if (this.sseConnection !== null) {
+  if (this.sseConnection) {
     this.sseConnection.close()
-    this.sseConnection = null
+    console.debug('SSE connection closed.')
```
[the remark this answers is not in the document]
Comment (b3n4kh) [reply src/gui/src/utils/sse.js:56]: I think this is a valid point - I don't think this would work

**Review by b3n4kh — CHANGES_REQUESTED** [review]

**Review by b3n4kh — APPROVED** [review]

**Pull request merged**